rs/nervous_system/integration_tests/src/pocket_ic_helpers.rs

```diff
@@ -733,7 +734,7 @@ pub async fn upgrade_nns_canister_to_tip_of_master_or_panic(
 /// ```
 pub async fn await_with_timeout<'a, T, F, Fut>(
     pocket_ic: &'a PocketIc,
-    timeout_seconds: u64,
+    expected_event_time: Range<u64>,
```

Comment:
```suggestion
    expected_event_interval_seconds: Range<u64>,
```

Reply:
Please also add a note in the doc string to explain what this does.

Reply:
Done!